.bundler-audit.yml

```diff
@@ -0,0 +1,4 @@
+---
+ignore:
+  - CVE-2021-41136
```

Comment:
filed https://github.com/MAAP-Project/mmt/issues/7 for these two issues (caused by puma)